File: Data Science/Assignment 3.py
import pandas as pd

# Load dataset
url = "https://raw.githubusercontent.com/mwaskom/seaborn-data/master/iris.csv"

df = pd.read_csv(url)

# Create Average Score
df["Average Score"] = (
    df["sepal_length"] +
    df["sepal_width"] +
    df["petal_length"] +
    df["petal_width"]
) / 4

# Sort data by Average Score (Ascending)
sorted_df = df.sort_values(by="Average Score", ascending=True)

print(sorted_df.head())
print(sorted_df.head())

File: Data Science R Language/assignment3.R
# Assignment 3

# Load dataset
url <- "https://raw.githubusercontent.com/mwaskom/seaborn-data/master/iris.csv"

df <- read.csv(url)

# Create Average Score
df$Average_Score <- (
  df$sepal_length +
  df$sepal_width +
  df$petal_length +
  df$petal_width
) / 4

# Sort dataset in ascending order
sorted_data <- df[order(df$Average_Score), ]

# Display first 10 rows
print(head(sorted_data, 10))
